bin/test

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+main() {
+    # parse the arguments into variables
+    declare build_dir="${1:-}"
+
+    # check if composer tests are available
+    if composer run-script -l 2>/dev/null | grep -q '^\s*test\b'; then
+        # cd into the build directory and run the tests via composer
+        cd "$build_dir" && composer test
+    else # if no tests are defined...
+        # ...display a friendly reminder to define some
+        printf "\033[0;31mNo test command found! Please define a test command in the composer.json in order to run test.\033[0m\n"
```

Comment:
It needs to be discussed whether this should result in an error or not, given how users have to enable CI explicitly.

Reply:
The `nodejs` buildpack doesn't check if there is a test command, so maybe this buildpack shouldn't either.

Reply:
But that results in `npm` printing an error, yet still exiting with code 0. Other languages behave differently. And the underlying Heroku CI system will error the run (not fail it) if no tests are defined (and the buildpack doesn't have default test run behavior).

Reply:
I think its worth just relying on the underlying toolchain. Its fairly easy for a user to set their test command to `true` if they decide that they do not want tests from their php buildpack to be invoked.

Reply:
The check for the `composer test` command should be in done in order to not break existing CI pipelines. This warning may be irrelevant, but notifies the user to be able to test with `composer test`. But I can remove it, if you don't like the notification.

Reply:
if that error messages prevents this pr from beeing merged, than just remove it, please @dj95